drivers/vfio/pci/vfio_pci_rdwr.c

```diff
@@ -202,7 +202,7 @@ static ssize_t do_io_rw(struct vfio_pci_device *vdev, bool test_mem,
 
 static int vfio_pci_setup_barmap(struct vfio_pci_device *vdev, int bar)
```

Comment:
`vdev->vdev` doesn't look good and doesn't represent the semanics

Would be nice to differentiate VFIO PCI from VFIO
```suggestion
static int vfio_pci_setup_barmap(struct vfio_pci_device *vpci_dev, int bar)
```